code/game/objects/effects/portals.dm

```diff
@@ -221,3 +221,16 @@ var/list/portal_cache = list()
 		close()
 
 /obj/effect/portal/wormhole/on_update_icon()
+
+/*
+Portal used to move from the ship to the junk field generated by junk tractor beam
+Basically a portal without time limit and failchance
+*/
+/obj/effect/portal/jtb
+	name = "junk field portal"
+	desc = "A portal stabilized by heavy-duty machinery. It is safe to cross."
+	failchance = 0
+	entropy_value = 0
```

Comment:
Entropy is fun, give it some entropy.

Reply:
@TheShown911 @Wouju The problem is that if I put an entropy value you can be sure a bonobo player will spend 15 minutes going back an forth between the junk field and the ship just to make the entropy skyrocket.

Reply:
People can already do that with moebius and IH teleporters.